packages/compass/src/app/components/workspace.tsx

```diff
@@ -67,6 +67,29 @@ export default function Workspace({
   const multiConnectionsEnabled = usePreference(
     'enableNewMultipleConnectionSystem'
   );
+  const renderSidebar = useCallback(
+    () => (
+      <CompassSidebarPlugin
+        singleConnectionConnectionInfo={singleConnectionConnectionInfo}
+      />
+      // This will be gone once we merge my-queries PR
+      // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
This is a scary way to break the rules of hooks especially when connection info is what's being passed here. Why is this required?

Reply:
this was a left over and I was supposed to remove this, thank you for catching up on this.

Reply:
We removed these changes now